src/routes/dashboard.svelte

```diff
@@ -16,9 +33,9 @@
   <link rel="stylesheet" href="/css/page-components/empty-state.css" />
 </svelte:head>
 
-<Layout title="Dashboard" adminsOnly>
+<Layout user={account}>
   <h1 class="padded">Dashboard</h1>
-  <div class="cards padded">
+  <div class="cards padded material">
```

Comment:
`div.material` should be wrapping `h1.padded` and `div.cards.padded`, as per the markup